utils/spacewalk-common-channels.ini

```diff
@@ -1199,95 +1199,95 @@ repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/UEKR4/latest/%(arch)s/
 
 [oraclelinux6-mysql55]
 label    = %(base_channel)s-mysql55
 archs    = %(_x86_archs)s
 name     = MySQL 5.5 for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/MySQL/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/MySQL/%(arch)s/
 
 [oraclelinux6-mysql56]
 label    = %(base_channel)s-mysql56
 archs    = %(_x86_archs)s
 name     = MySQL 5.6 for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/MySQL56/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/MySQL56/%(arch)s/
 
 [oraclelinux6-mysql57]
 label    = %(base_channel)s-mysql57
 archs    = %(_x86_archs)s
 name     = MySQL 5.7 for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux7-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/MySQL57_community/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/MySQL57_community/%(arch)s/
 
 [oraclelinux6-playground]
 label    = %(base_channel)s-playground
 archs    = x86_64
 name     = Playground (Mainline) Kernels for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/playground/latest/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/playground/latest/%(arch)s/
 
 [oraclelinux6-spacewalk22-server]
 label    = %(base_channel)s-spacewalk22-server
 archs    = x86_64
 name     = Spacewalk 2.2 Server for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/spacewalk22/server/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/spacewalk22/server/%(arch)s/
 
 [oraclelinux6-spacewalk22-client]
 label    = %(base_channel)s-spacewalk22-client
 archs    = %(_x86_archs)s
 name     = Spacewalk 2.2 Client for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/spacewalk22/client/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/spacewalk22/client/%(arch)s/
 
 [oraclelinux6-spacewalk24-server]
 label    = %(base_channel)s-spacewalk24-server
 archs    = x86_64
 name     = Spacewalk 2.4 Server for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/server/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/server/%(arch)s/
 
 [oraclelinux6-spacewalk24-client]
 label    = %(base_channel)s-spacewalk24-client
 archs    = %(_x86_archs)s
 name     = Spacewalk 2.4 Client for Oracle Linux 6 (%(arch)s)
 base_channels = oraclelinux6-%(arch)s
-repo_url = http://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/client/%(arch)s/
+repo_url = https://yum.oracle.com/repo/OracleLinux/OL6/spacewalk24/client/%(arch)s/
 
 [oraclelinux6-scl]
```

Comment:
Can also be changed to https.

Reply:
What do you mean :-?

I see the diff and it's now `https`.

Reply:
Ah, saw it now. You mean `oraclelinux6-scl` itself. Sorry.

Reply:
Yes, I could not add the comment at that place